### PR TITLE
[2015.2] lithium deprecations

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

**Comment by s0undt3ch** [comment]
@cachedout I ended up splitting the PR, mixing both with the current _stability_ of jenkins wasn't a good idea.

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

File: doc/topics/releases/2015.2.0.rst [hunk 1]
```diff
@@ -11,3 +11,22 @@ Salt SSH
...
+Deprecations
+============
+- Removed ``parameter`` keyword argument from ``eselect.exec_action`` execution
+  module.
+
+- Removed ``runas`` parameter from the following ``pip``` execution module
+  functions: ``install``, ``uninstall``, ``freeze``, ``list_``, ``list_upgrades``,
+  ``upgrade_available``, ``upgrade``. Please migrate to ``user``.
+
+- Removed ``runas`` parameter from thge following ``pip`` state module
```
Comment (joehoyle) [inline doc/topics/releases/2015.2.0.rst]: typo `thge ` here

**Comment by techhat** [comment]
Outside of the typo, looks good to me.

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

**Comment by s0undt3ch** [comment]
@thatch45, @cachedout: This is finally ready.

**Comment by rallytime** [comment]
Look at that green! :)

**Comment by s0undt3ch** [comment]
:)

Please review this carefully since I'm **consuming** the load/low/chunks...

**Comment by thatch45** [comment]
Those load pops should all be good! Thanks @s0undt3ch !!!

**Pull request merged**